catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlAction.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.cql;
+
+import ddf.action.Action;
+import ddf.action.impl.ActionImpl;
+
+public class CqlAction extends ActionImpl {
```

Comment:
Why did you name this CqlAction? My understanding was that you are trying to rename some of the export display names. My gut feeling is that there is probably a better name for this.

Reply:
All of the classes in this package are prepended with `Cql` and since I'm extending the functionality of an action I figured `CqlAction` would be appropriate.

Reply:
`CqlAction` is a bit strange because this action has nothing to do with CQL. Maybe `DisplayableAction` since this adds a display name?